javascript/react/src/verbs/Convert.base.tsx

```diff
@@ -54,8 +64,24 @@ export const ConvertBase: React.FC<
 					(s.args.formatPattern = value ? value : '%Y-%m-%d'),
 			},
 		],
-		[step, columns],
+		[step, columns, columnsMetadata],
 	)
 
 	return <VerbForm inputs={inputs} step={step} onChange={onChange} />
 })
+
+function isInputColumnArray(columnsMetadata: ColumnMetadata[], columns: string[]){
+	let arrayFlag = false
+	let index = 0
+
+	while(!arrayFlag && index < columns.length){
```

Comment:
We can simplify this by removing the column lookup and using Array.some (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

e.g.
```ts
return columnsMetadata.some((metadata) = metadata.type === DataType.Array)
```

I don't thinking looking up the column name is doing much

Reply:
It is needed by column, because the component receives all column names and we need to enable the array delimiter just if one column with array datatype was selected into the multidropdown.

Reply:
gotcha, maybe we can filter down the metadata list first then?